docker/Dockerfile.hdfs

```diff
@@ -21,6 +21,12 @@ ENV HADOOP_COMMON_LIB_NATIVE_DIR $HADOOP_INSTALL/lib/native
 ENV HADOOP_OPTS "-Djava.library.path=$HADOOP_INSTALL/lib/native"
 ENV LD_LIBRARY_PATH $LD_LIBRARY_PATH:$JAVA_HOME/jre/lib/amd64/server
 
+#Set your nameserver info in the docker image
```

Comment:
Ok,my first run failed just because the container instance could not resolve the DNS so I added these two lines,anyway it's ok to make them as comments.The comments may be also helpful when some guy wanna to use unified containerizer in mesos with this docker image.